specification/source/glossary.rst

```diff
@@ -0,0 +1,50 @@
+.. _sec_glossary:
+
+Glossary
+--------
+
+This section provides definitions for abbreviations and terms introduced in
+the Open Building Controls project.
+
+.. If you add new entries, keep the alphabetical sorting.
+
+.. glossary::
+
+   Controls Description Language
+     The Control Description Language (CDL) is the language
+     that is used to express control sequences and requirements.
```

Comment:
This seems worth adding to the glossary. I can add it on the master if you agree.